src/ControlFlow/Node/Expr/BinaryOp/Equal.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @author Patsura Dmitry https://github.com/ovr <talk@dmtry.me>
+ */
+
+namespace PHPSA\ControlFlow\Node\Expr\BinaryOp;
+
+use PHPSA\ControlFlow\Node\AbstractNode;
+
+class Equal extends AbstractNode
```

Comment:
maybe extend a new abstract node called "BinaryOpNode" that has $left, $right, $result and getSubVariables?
It's the same for all binary ops it seems or is this gonna change later?